src/plugins/gr-he-common/core/shell.py

```diff
@@ -103,5 +109,34 @@ def _setup(self):
                 if not self.environment[ohostedcons.CoreEnv.TMUX_PROCEED]:
                     raise context.Abort('Aborted by user')
 
+        if (
+            not self.environment[ohostedcons.NetworkEnv.FORCE_IPV4]
+            and not self.environment[ohostedcons.NetworkEnv.FORCE_IPV6]
+        ):
+            dialog.queryEnvKey(
+                dialog=self.dialog,
+                logger=self.logger,
+                env=self.environment,
+                key=ohostedcons.CoreEnv.FORCE_IP_PROCEED,
+                name='FORCE_IP_PROCEED',
+                note=_(
+                    '\nIt has been detected that either --4 or --6 '
+                    'options were not used.\n'
+                    'On a dual stack environment, IPv6 will be selected.\n'
+                    'If you want to use IPv6, make sure that DNS returns '
+                    'only IPv6.\n'
```

Comment:
I'd rather say something like:

Dual stack IPv4/IPv6 is not supported, and might or might not work.

(Instead of all of lines 125-127).

Reply:
Very well.

My point is:

I am not sure we support equally well all the possible combinations of "dual stack". If we want to officially support dual stack, by all means - just write about the known limitations and fix whatever new reports that come in. I never understood so far that this is your point.